src/routes/posts.js

```diff
@@ -149,6 +148,71 @@ const fetchPosts = async (req, res, next) => {
 	}
 }
 
+// ASSIGN TAG TO AN ARTICLE
+
+const assignTagToPost = async (req, res, next) => {
+	try {
+		const { postId } = req.params // post id;
+		const { tagId } = req.body
+
+		const postsTags = await db.query(
+			// eslint-disable-next-line max-len
+			'INSERT INTO posts_tags ("postId","tagId") SELECT $1,$2 WHERE NOT EXISTS (SELECT * FROM posts_tags WHERE "postId" = $1 AND "tagId" = $2) RETURNING *',
```

Comment:
@Yellow-May, i generally think our queries should be broken down on different lints to make other devs at least process it a bit better. 

so i think the eslint max len helps and should not be disabled. 

So mostly we can start the newline with the SQL keyword like we see in certain articles. The backtick (`) will help with the multiline as you might already know

```sql
	INSERT INTO posts_tags ("postId","tagId")
	SELECT $1,$2 
	WHERE NOT EXISTS (SELECT * FROM posts_tags WHERE "postId" = $1 AND "tagId" = $2)
	RETURNING *,
```

Reply:
Yeah Myself and @jakazzy were working on it yesterday
However because it is a single string, we cannot break it into different lines

There are many other instances in other files because of the long queries, they couldn't be wrapped into new lines

Reply:
just came to mind that for this, i think moving forward we could just make the combination of both (tagId,postId) unique so that the database will naturally let us know the tag already exists and we don't have to make another SELECT in the insert.

otherwise a manual insert into the db will get no such error

Reply:
> There are many other instances in other files because of the long queries, they couldn't be wrapped into new lines

oh okay, even with a backtick character (`). that won't work

Reply:
we have an example for our `getPost` function here where we use the backtick character

https://github.com/tbf-challenge/teamwork-be/blob/62226a075b9ecd0887286a0c0b38265c0ac8cab9/src/routes/posts.js#L87-L92

Reply:
> just came to mind that for this, i think moving forward we could just make the combination of both (tagId,postId) unique so that the database will naturally let us know the tag and post already exists and we don't have to make another SELECT in the insert.
> 
> otherwise a manual insert into the db will get no such error

Yeah that would be better

Reply:
> > There are many other instances in other files because of the long queries, they couldn't be wrapped into new lines
> 
> oh okay, even with a backtick character (`)?. that won't work?

Yeah i tried that as well, didn't work
We were going to consider interpolation but we didn't go through with it

Reply:
> Yeah i tried that as well, didn't work
We were going to consider interpolation but we didn't go through with it

okay cool, no problem, thanks. Should i go ahead with the merge then?

Reply:
Yes